src/frontend/api/fagsak.ts

```diff
@@ -83,19 +88,21 @@ export interface IRestVilkårsvurdering {
 export const apiOpprettEllerOppdaterVedtak = (fagsakId: number, data: IRestVilkårsvurdering) => {
     return axiosRequest<IFagsak>({
         data,
-        method: 'PUT',
-        url: `/familie-ba-sak/api/fagsak/${fagsakId}/vedtak`,
+        method: 'POST',
+        url: `/familie-ba-sak/api/fagsaker/${fagsakId}/vedtak`,
     });
 };
 
 export interface IOpprettBeregningData {
     barnasBeregning: IBarnBeregning[];
 }
 
-export const apiOpprettBeregning = (fagsakId: number, data: any) => {
+export const apiOpprettBeregning = (fagsak: IFagsak, data: any) => {
+    const vedtakId = aktivVedtak(fagsak)?.id;
```

Comment:
Should we maybe pass these id's into the functions? If things fail here we don't have any nice ways to fail. In the frontend we present aktivVedtak, so we can pass that value to the function (for instance)